runhouse/main.py

```diff
@@ -595,27 +593,94 @@ def status(
+    table = Table(title=table_title)
+
+    # Add columns to the table
+    table.add_column("Name", justify="left", no_wrap=True)
+    table.add_column("Cluster Type", justify="left", no_wrap=True)
+    table.add_column("Status", justify="left")
+    table.add_column("Last Active (UTC)", justify="left")
+
+    return table
+
 
-    get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
-    clusters_in_den_resp = await httpx_client.get(
-        f"{rns_client.api_server_url}/resource",
-        params=get_clusters_params,
-        headers=rns_client.request_headers(),
+def _add_cluster_as_table_row(table: Table, rh_cluster: dict):
+    last_active = rh_cluster.get("Last Active (UTC)")
+    last_active = last_active if last_active != "1970-01-01 00:00:00" else "Unknown"
+    table.add_row(
+        rh_cluster.get("Name"),
+        rh_cluster.get("Cluster Type"),
+        rh_cluster.get("Status"),
+        last_active,
     )
 
-    return clusters_in_den_resp
+    return table
+
+
+def _add_clusters_to_output_table(
+    table: Table, clusters: list[Dict], filters_requested: dict = None
+):
+
+    for rh_cluster in clusters:
+        last_active_at = rh_cluster.get("Last Active (UTC)")
+        last_active_at_no_offset = str(last_active_at).split("+")[
+            0
+        ]  # The split is required to remove the offset (according to UTC)
+        rh_cluster["Last Active (UTC)"] = last_active_at_no_offset
+        rh_cluster["Status"] = get_status_color(rh_cluster.get("Status"))
+
+        if filters_requested:
+            table = _add_cluster_as_table_row(table, rh_cluster)
+        else:
+            # Default behaviour: Print Running clusters that were active it the last 24 hours
+            now = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
+            if (now - last_active_at).total_seconds() <= LAST_ACTIVE_AT_TIMEFRAME:
+                table = _add_cluster_as_table_row(table, rh_cluster)
+
+
+def _print_msg_live_sky_clusters(live_clusters_not_in_den: int):
+    if live_clusters_not_in_den > 0:
+        live_sky_clusters_str = (
+            f"There are {live_clusters_not_in_den} live clusters that are not saved in Den."
+            if live_clusters_not_in_den > 1
+            else "There is 1 live cluster that is not saved in Den."
+        )
+
+        clusters_pronoun = "them" if live_clusters_not_in_den > 1 else "it"
+
+        console.print(
+            f"{live_sky_clusters_str} To get information about {clusters_pronoun}, please run [bold italic]sky status -r[/bold italic]."
+        )
 
 
-def get_clusters_from_den():
-    return asyncio.run(aget_clusters_from_den())
+def supported_time_units():
+    return ", ".join(list(TIME_UNITS.keys()))
 
 
 @cluster_app.command("list")
-def cluster_list():
+def cluster_list(
+    get_all_clusters: bool = typer.Option(
+        False,
+        "--a",
```

Comment:
```suggestion
        "--all",
```
(for the long option)